CONTRIBUTING.md

```diff
@@ -6,24 +6,22 @@ Please fork, branch and pull-request any changes you'd like to make. For more in
 
 ## Workspace setup
 
-1. Clone this repository
+1. Clone your fork of the repository
 
-    `git clone https://github.com/azure/iotedgedev.git`
+    `git clone https://github.com/<your_id>/iotedgedev.git`
 
-2. Rename `.env.tmp` in the root of the repo to `.env` and set the `IOTHUB_CONNECTION_STRING` and `DEVICE_CONNECTION_STRING` values to settings from your IoT Hub and Edge Device. To set these values you could run `iotedgedev iothub setup` in the root of the repo.
-
-3. Install **[Docker](https://docs.docker.com/engine/installation/)**
+1. Install **[Docker](https://docs.docker.com/engine/installation/)**
```

Comment:
Nit: Mariana will mention that they'd prefer to have the numbers explicitly set in the readme, instead of only having them generated in the markdown preview.